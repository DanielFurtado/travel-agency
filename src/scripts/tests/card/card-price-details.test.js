import React from 'react';
import { shallow } from 'enzyme';
import CardPriceDetails from '../../../components/card/card-price-details';

describe('CardPriceDetails', () => {
  const cardPriceDetails = shallow(<CardPriceDetails />);
  
  it('renders properly', () => {
    expect(cardPriceDetails).toMatchSnapshot();
  });
});