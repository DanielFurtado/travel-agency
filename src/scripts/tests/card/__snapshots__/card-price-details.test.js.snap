// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardPriceDetails renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardPriceDetails />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <i
          className={undefined}
/>,
        <span
          className="card-price-text"
>
          
</span>,
      ],
      "className": "card-price-info",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "i",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
            undefined,
          ],
          "className": "card-price-text",
        },
        "ref": null,
        "rendered": Array [
          undefined,
          undefined,
        ],
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <i
            className={undefined}
/>,
          <span
            className="card-price-text"
>
            
</span>,
        ],
        "className": "card-price-info",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "i",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              undefined,
            ],
            "className": "card-price-text",
          },
          "ref": null,
          "rendered": Array [
            undefined,
            undefined,
          ],
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
